loveship/components/tables/columnsConfigurationModal/ColumnsConfigurationModal.tsx

```diff
@@ -22,10 +22,12 @@ interface ColumnsConfigurationModalProps<TItem, TId, TFilter> extends IModal<Col
     columns: DataColumnProps<TItem, TId, TFilter>[];
 }
 
-const renderGroupTitle = (title: string, amount: number) => <FlexRow size="24" padding="24" spacing="6" cx={ styles.groupTitle }>
-    <Text cx={ styles.groupTitleText } font="sans-semibold" lineHeight="24" fontSize="14">{ title }</Text>
-    <Badge cx={ styles.groupTitleBadge } caption={ amount } color="night300" size="18" />
-</FlexRow>;
+const renderGroupTitle = (title: string, amount: number, additionalCx?: CX) => (
```

Comment:
remove additionalCx, try to make it via css cascading

Reply:
Done.